tst/EnergyPlus/unit/MixedAir.unit.cc

```diff
@@ -5486,32 +5500,95 @@ TEST_F(EnergyPlusFixture, MechVentController_VRPNoCap)
 
     ASSERT_TRUE(process_idf(idf_objects));
 
-    Real64 SysMassFlow(0.0);        // System supply mass flow rate [kg/s]
-    Real64 OAMassFlow(0.0);         // OA mass flow rate [kg/s]
-    Real64 ExpectedOAMassFlow(0.8); // System design OA flow rate
-    bool ErrorsFound(false);
+    Real64 OAMassFlow = 0.0; // OA mass flow rate [kg/s]
+    bool ErrorsFound = false;
 
     GetZoneData(*state, ErrorsFound);
     GetOARequirements(*state);
     GetZoneEquipment(*state);
     GetOAControllerInputs(*state);
+    ZoneAirLoopEquipmentManager::GetZoneAirLoopEquipment(*state);
+    SingleDuct::GetSysInput(*state);
```

Comment:
I see, so you could call single duct for both TUs.

Reply:
> I see, so you could call single duct for both TUs.

And keep the connections simple, and not need a fan object, etc.